uk.co.compucorp.civicrm.hrcontactactionsmenu/templates/CRM/HRContactActionsMenu/Page/Inline/ActionsPart2.tpl

```diff
@@ -1,122 +1,127 @@
-<div id="crm-contact-actions-list">
-  <div id="bootstrap-theme">
-    <div class="row">
-      <div class="col-md-4 crm_contact_action_menu__user_info">
-        <div class="row">
-          <h3>User Informations:</h3>
-          <p><span class="crm_contact_action_menu__bold_text">User:</span> <a href="#" class="text-primary"> 100 Lauren Ipsum </a></p>
-          <p><span class="crm_contact_action_menu__bold_text">Roles:</span> <span class="crm_contact_action_menu__grey_text"> civihr_staff, civihr_manager </span></p>
-          <div class="crm_contact_action_menu__bottom_margin">
-            <button class="btn btn-primary-outline">
-              <span>
-                <i class="fa fa-envelope-o"></i>
-              </span>
-              Send Welcome Email
-            </button>
-          </div>
-          <div class="crm_contact_action_menu__bottom_margin">
-            <button class="btn btn-primary-outline">
-              <span>
-                <i class="fa fa-envelope-o"></i>
-              </span>
-              Send Password Reset Email
-            </button>
-          </div>
-          <h3>User Has Access To:</h3>
-          <p>
-            <span class="crm_contact_action_menu__grey_text">No staff members</span>
-            <span class="fa fa-question-circle crm_contact_action_menu__help_icon"></span>
-          </p>
-          <button class="btn btn-secondary-outline">Manage Regional Access</button>
-        </div>
-        <div class="row crm_contact_action_menu__control_buttons_container">
-          <hr>
-          <div class="col-md-6">
-            <button class="button__delete btn-warning">Disable User Account</button>
-          </div>
-          <div class="col-md-6">
-            <button class="button__disable btn-danger">Delete User Account</button>
+<div id="crm-contact-actions-wrapper" data-edit-params='{ldelim}"cid": "{$contactId}", "class_name": "CRM_Contact_Page_Inline_Actions"{rdelim}'>
+  {crmButton id="crm-contact-actions-link" href="#" icon="bars"}
+    {ts}Actions{/ts}
+  {/crmButton}
+  <div id="crm-contact-actions-list" class="crm_contact-actions">
+    <div id="bootstrap-theme">
+      <div class="crm_contact-actions__body">
+        <div class="col-md-4 crm_contact-actions__body__user-info crm_contact-actions__user-info--margin">
+          <div class="col-md-12">
+            <h3>User Informations:</h3>
+            <p><span class="crm_contact-actions--bold">User:</span> <a href="#" class="text-primary"> 100 Lauren Ipsum </a></p>
+            <p><span class="crm_contact-actions--bold">Roles:</span> <span class="crm_contact-actions--dark-grey"> civihr_staff, civihr_manager </span></p>
+            <div class="crm_contact-actions--bottom-margin">
```

Comment:
what's this class for? Please add screenshot

Reply:
Replied above!